jhsingle_native_proxy/main.py

```diff
@@ -169,9 +169,12 @@ def get_port_from_env():
 @click.option('--progressive/--no-progressive', default=False, help='Progressively flush responses as they arrive (good for Voila)')
 @click.option('--websocket-max-message-size', default=0, type=click.INT, help='Max size of websocket data (default 0, meaning leave to library defaults)')
 @click.argument('command', nargs=-1, required=True)
+@click.option('--aiohttp-no-ssl-connector', is_flag=True, help='Sets up pre-configured TCP connector. Include to disable SSL verification')
```

Comment:
We need an explicit default option here

Reply:
Unless the flag does it by default

Reply:
Hi Konstantin, the flag does it by default.